[Doc Improvement][Liveshare permission update][3772666] (#10788)
MicrosoftDocs/msteams-docs: msteams-platform/resources/schema/manifest-schema-dev-preview.md

    |`MeetingStage.Write.Chat`|Allows the app to show content on the meeting stage in meetings associated with the chat, on behalf of the signed-in user.|
    |`OnlineMeetingParticipant.Read.Chat`|Allows the app to read participant information, including name, role, id, joined, and left times, of meeting associated with the chat, on behalf of the signed-in user.|
    |`OnlineMeetingParticipant.ToggleIncomingAudio.Chat`|Allows the app to toggle incoming audio for participants in meetings associated with the chat, on behalf of the signed-in user.|

>     |`LiveShareSession.ReadWrite.Chat`|Allows the app to create and synchronize Live Share sessions for the chat. Provide access related information about the meeting's roster, such as member's meeting role, on behalf of the signed-in user.|

Reviewer: For meetings associated with the chat, also provide access related ....

Reviewer: Please apply this consistent across all the strings; otherwise, after the update, in its current form, the second sentence is a dangler with no relation to the first sentence.

Author: Updated the doc as suggested

    |`ChannelMeetingAudioVideo.Stream.Group`| Allows the app to stream audio-video content from channel meetings associated with the team. |
    |`InAppPurchase.Allow.Group`| Allows the app to show marketplace offers to users in the team and complete their purchases within the app, on behalf of the signed-in user.|
    |`ChannelMeetingStage.Write.Group`| Allows the app to show content on the meeting stage in channel meetings associated with the team, on behalf of the signed-in user.|

>     |`LiveShareSession.ReadWrite.Group`|Allows the app to create and synchronize Live Share sessions for the team and provide access related information about the team's roster, such as name and meeting role, on behalf of the signed-in user.|

Reviewer: provide access related information, such as name and role, about the team's roster and any associated meetings, on behalf of the signed-in user.

Author: Updated as suggested

>     |`LiveShareSession.ReadWrite.Group`|Allows the app to create and synchronize Live Share sessions for the team and for any meetings associated with the team, provide access related information about the meeting's roster, such as member's meeting role, on behalf of the signed-in user.|

Reviewer 2: ```suggestion
    |`LiveShareSession.ReadWrite.Group`|Allows the app to create and synchronize Live Share sessions for the team and provide access related information about the team's roster, such as name and meeting role, on behalf of the signed-in user.|
```

    |`MeetingStage.Write.Chat`|Allows the app to show content on the meeting stage in meetings associated with the chat, on behalf of the signed-in user.|
    |`OnlineMeetingParticipant.Read.Chat`|Allows the app to read participant information, including name, role, id, joined, and left times, of meeting associated with the chat, on behalf of the signed-in user.|
    |`OnlineMeetingParticipant.ToggleIncomingAudio.Chat`|Allows the app to toggle incoming audio for participants in meetings associated with the chat, on behalf of the signed-in user.|

>     |`LiveShareSession.ReadWrite.Chat`|Allows the app to create and synchronize Live Share sessions for the chat and provide access related information about the chat's roster, such as name and meeting role, on behalf of the signed-in user.|

Reviewer: provide access related information, such as name and role, about the chat's roster and any associated meetings, on behalf of the signed-in user.

Author: Updated as suggested

>     |`LiveShareSession.ReadWrite.Chat`|Allows the app to create and synchronize Live Share sessions for the chat and for any meetings associated with the chat, provide access related information about the meeting's roster, such as member's meeting role, on behalf of the signed-in user.|

Reviewer 2: ```suggestion
    |`LiveShareSession.ReadWrite.Chat`|Allows the app to create and synchronize Live Share sessions for the chat and provide related information about the chat's roster, such as name and meeting role, on behalf of the signed-in user.|
```
